### PR TITLE
chore: Update outdated dependencies

### PR DESCRIPTION
**Pull request by manoj9788**
Shall look for the Travis log ..
Resolving #3249

### CODE REVIEW

File: typings.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+    "adm-zip": "registry:dt/adm-zip#0.0.0+20160317120654",
     "chalk": "registry:dt/chalk#0.4.0+20160317120654",
     "form-data": "registry:dt/form-data#0.0.0+20160316155526",
-    "glob": "registry:dt/glob#5.0.10+20160317120654",
-    "minimatch": "registry:dt/minimatch#2.0.8+20160317120654",
-    "node": "registry:dt/node#4.0.0+20160412142033",
-    "optimist": "registry:dt/optimist#0.0.0+20160316171810",
+    "jasmine": "registry:dt/jasmine#2.2.0+20160412134438",
+    "minimist": "registry:dt/minimist#1.1.3+20160317120654",
+    "node": "registry:dt/node#4.0.0+20160423143914",
```
Comment (cnishina) [inline typings.json]: Add back `glob`, `minimatch`, and `optimist`. Remove `jasmine` and `minimist`. When running `typings install` if the typings dependencies are out of date, please update them.

File: typings.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-  "ambientDependencies": {
+  "globalDependencies": {
+    "adm-zip": "registry:dt/adm-zip#0.0.0+20160317120654",
```
Comment (cnishina) [inline typings.json]: remove `adm-zip`

File: package.json [hunk 3]
```diff
@@ -14,33 +14,33 @@
...
+    "jshint": "2.9.2",
     "lodash": "^4.5.1",
     "marked": "^0.3.3",
-    "mocha": "2.3.4",
+    "mocha": "2.5.3",
     "rimraf": "~2.5.0",
     "run-sequence": "^1.1.5",
-    "typescript": "1.8.2",
-    "typings": "^0.8.1"
+    "typescript": "1.8.10",
```
Comment (cnishina) [inline package.json]: use "^1.8.10"

File: package.json [hunk 4]
```diff
@@ -14,33 +14,33 @@
...
+    "typings": "1.0.4"
```
Comment (cnishina) [inline package.json]: use "^1.0.4"

File: package.json [hunk 5]
```diff
@@ -14,33 +14,33 @@
...
+    "body-parser": "1.15.1",
+    "chai": "~3.5.0",
+    "chai-as-promised": "~5.3.0",
     "clang-format": "^1.0.34",
     "expect.js": "~0.3.1",
     "express": "~4.13.3",
     "gulp": "^3.9.1",
     "gulp-clang-format": "^1.0.23",
-    "jshint": "2.9.1",
+    "jshint": "2.9.2",
```
Comment (cnishina) [inline package.json]: use "^2.9.2"

File: package.json [hunk 6]
```diff
@@ -14,33 +14,33 @@
   "dependencies": {
     "adm-zip": "0.4.7",
     "chalk": "^1.1.3",
-    "glob": "~6.0",
+    "glob": "~7.0.3",
```
Comment (cnishina) [inline package.json]: use "^7.0.3"

File: tsconfig.json [hunk 7]
```diff
@@ -12,10 +12,6 @@
   "exclude": [
     "built",
     "node_modules",
-    "testapp",
-    "typings/browser",
-    "typings/browser.d.ts",
-    "typings/main",
-    "website"
+    "typings/globals"
```
Comment (cnishina) [inline tsconfig.json]: You need to add back in 'testapp' and 'website'. If not, tsc will try to do transpile the website and the testapp directory

File: tsconfig.json [hunk 8]
```diff
@@ -17,5 +17,6 @@
     "typings/browser.d.ts",
     "typings/main",
     "website"
```
Comment (cnishina) [inline tsconfig.json]: add a comma after "website"

File: tsconfig.json [hunk 9]
```diff
@@ -17,5 +17,6 @@
     "typings/browser.d.ts",
     "typings/main",
```
Comment (cnishina) [inline tsconfig.json]: remove:
```
    "typings/browser",
    "typings/browser.d.ts",
    "typings/main",
```

**Comment by cnishina** [comment]
This appears to also break `node scripts/interactive_tests/interactive_test.js` where the client is connecting to the server. It would be nice if this could also be looked into.

**Comment by manoj9788** [comment]
@cnishina finally I see the Travis build is green. lets go through the logs and advice if any changes.

**Comment by cnishina** [comment]
@manoj9788 Did some research and I did not update the debugger which was failing the `interactive_tests`. LGTM.

**Comment by manoj9788** [comment]
@cnishina thanks for clarifying and merging.

**Pull request merged**